### ISSUE TITLE
Large redwoods are generating without quarter logs or roots

### ISSUE THREAD

**Issue by saltyseadoggo** [TerraformersMC/Terrestria#246]
The 2x2 redwood trees in the Redwood Forest biome are generating made of regular redwood logs instead of quarter logs, and without roots as well. They do not match the redwoods generated from saplings, which do have those features.

Steps to Reproduce:
- Start Minecraft with Terrestria and the Fabric API loaded.
- Create a new single biome world with the biome set to terrestria:redwood_forest.
- Look around at the rootless, non-quarter-logged trees.

Attachments:
![2021-03-01_13 48 48](https://user-images.githubusercontent.com/69029346/109565888-f391c880-7a97-11eb-9290-5dc98246d417.png)
![2021-03-01_13 48 54](https://user-images.githubusercontent.com/69029346/109565895-f55b8c00-7a97-11eb-8f31-5686e5a3ee38.png)

**Comment by NeusFear** [comment]
Does this happen in the hemlock forests too?

**Comment by saltyseadoggo** [comment]
Yes. I created a single biome world set to terrestria:hemlock_rainforest, and the hemlocks generated had non-quarter logs and no roots as well.
